functions/pubsub/main.py

```diff
@@ -62,7 +61,7 @@ def publish(request):
 
 # [START functions_pubsub_subscribe]
 # Triggered from a message on a Cloud Pub/Sub topic.
-def subscribe(event, context):
+def subscribe(event):
```

Comment:
So why did we have `context` argument in the first place? Isn't there a case the functions service is sending the 2nd argument?

Reply:
**Update:** GCF itself sends `context` as a second positional argument, so we _cannot_ ignore it in the method signature.